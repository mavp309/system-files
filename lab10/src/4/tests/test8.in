5000000 3456789 123567898 798678954
999999999 533689999 445677888
