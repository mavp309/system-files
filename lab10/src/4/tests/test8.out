5000001003456788657257898244356842
